provisioners/src/main/java/org/jboss/intersmash/provision/operator/OpenDataHubOperatorProvisioner.java

```diff
@@ -105,20 +112,25 @@ public void undeploy() {
 				.level(Level.DEBUG)
 				.waitFor();
 
-		if (getApplication().getDSCInitialization() != null) {
-			final String appName = getApplication().getName();
-			deletionDetails = dscInitializationClient().withName(appName).delete();
-			deleted = deletionDetails.stream().allMatch(d -> d.getCauses().isEmpty());
-			if (!deleted) {
-				log.warn("Wasn't able to remove the 'DSCInitialization' resources created for '{}' instance!",
-						appName);
-			}
-			new SimpleWaiter(() -> dscInitializationClient().list().getItems().isEmpty()).level(Level.DEBUG).waitFor();
+		deletionDetails = dscInitializationClient().withName(appName).delete();
+		deleted = deletionDetails.stream().allMatch(d -> d.getCauses().isEmpty());
+		if (!deleted) {
+			log.warn("Wasn't able to remove the 'DSCInitialization' resources created for '{}' instance!",
```

Comment:
have you checked if this warning is logged or not?

Reply:
Actually not logged, since the removal works.